invoicer_project/invoices/invoice.py

```diff
@@ -0,0 +1,108 @@
+from io import BytesIO
+from reportlab.pdfgen import canvas
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from reportlab.lib.pagesizes import letter
+def create_pdf(serializer, items):
```

Comment:
please, follow the pep8 code conventions. Add spaces before function definition

Reply:
I suggest you to use r[eformat code in pycharm ](https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html)

Reply:
Done